include/odp/api/spec/ipsec.h

```diff
@@ -772,6 +772,76 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/** IPv4 address size */
+#define ODP_IP4ADDR_SIZE 4
```

Comment:
These could be defined in a new file (odp/api/spec/protocols.h).  ODP_IPV4_ADDR_SIZE, ODP_IPV6_ADDR_SIZE. So, that naming convention for IPv4 would be ODP_IPV4_XXX.

Reply:
> 
> 
> These could be defined in a new file (odp/api/spec/protocols.h). ODP_IPV4_ADDR_SIZE, ODP_IPV6_ADDR_SIZE. So, that naming convention for IPv4 would be ODP_IPV4_XXX.

With V8, added separate commit adding protocols.h. Also as suggested, renamed the macros to ODP_IPV4_ADDR_SIZE and ODP_IPV6_ADDR_SIZE.

Let me know if the API spec for SA info looks fine now. I can continue improving on the linux-generic implementation if you can confirm the spec.